testsuite/classlibrary/TestNoOps.sc

```diff
@@ -78,28 +85,34 @@ TestNoOps : UnitTest {
 		^if(item.isSequenceableCollection) {
 			item.collect(this.deepEval(_))
 		} {
-			item.value
+			item.value.value // for combinations of functions and Operands call value of the value
 		}
 
 	}
 
 	assertEqualValue { |a, b, comment, report = true, onFailure|
 		var diff, size;
-		a = a.value;
-		b = b.value;
+
+		a.postln;
```

Comment:
Because then you can explain what `a` means or, to put it another way, you can avoid adding an extra line of unexplained output to test results.